gems/sorbet-runtime/test/types/types.rb

```diff
@@ -212,13 +212,10 @@ def subtype_of?(other)
         assert_equal(Integer, type.type.raw_type)
       end
 
-      it 'fails if an element of the array is the wrong type' do
+      it 'does not fail if an element of the array is the wrong type' do
         type = T::Array[Integer]
         value = [true]
-        msg = type.error_message_for_obj(value)
-        expected_error = "Expected type T::Array[Integer], " \
-                         "got T::Array[T::Boolean]"
-        assert_equal(expected_error, msg)
+        assert_nil(type.error_message_for_obj(value))
```

Comment:
@aisamanra What does it mean for `error_message_for_obj` to be `nil`? Do we still show an "expected / actual" error message?

Reply:
Oh I see, these are all saying that `error_message_for_obj` is `nil` because there is no error raised, because the only problem is that the element type is mismatched, not the class type.